.github/ISSUE_TEMPLATE/documentation_bug.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Documentation style bug
+about: Report documentation that has lost its style
```

Comment:
```suggestion
name: Documentation bug
about: Report a problem with a specific documentation
```

Since the problem could also be:

- documentation is missing a page
- documentation is missing a sidebar entry
- ...?